administration/src/mui-modules/application/forms/WorkAtOrganizationForm.tsx

```diff
@@ -99,7 +99,7 @@ const WorkAtOrganizationForm: Form<State, Options, ValidatedInput, AdditionalPro
       <SubForms.payment.Component
         state={state.payment}
         setState={useUpdateStateCallback(setState, 'payment')}
-        label='Für diese ehrenamtliche Tätigkeit wurde eine Aufwandsentschädigung gewährt.'
+        label='Für diese ehrenamtliche Tätigkeit wurde eine Aufwandsentschädigung gewährt, die über den jährlichen Freibetrag hinaus geht. (840 Euro Ehrenamtspauschale bzw. 3000 Euro Übungsleiterpauschale)'
```

Comment:
@michael-markl i really couldn't figure out why i get a ts error by just editing the label. It shouldn't be a map problem in the backend or anything
```js
Argument type {workSinceDate: {maximumDate: null}, amountOfWork: {min: number, max: number}, payment: {readonly required: false}} is not assignable to parameter type SubFormsOptions<{responsibility: Form<State, Options, ValidatedInput, AdditionalProps>, organization: Form<State, Options, ValidatedInput, AdditionalProps>, workSinceDate: Form<State, {maximumDate: Date, maximumDateErrorMessage: string} | {maximumDate: null}, ValidatedInput, AdditionalProps>, amountOfWork: Form<State, Options, number, AdditionalProps>, certificate: Form<{MIMEType: keyof typeof defaultExtensionsByMIMEType, arrayBufferKey: number, filename: string} | null, Options, ValidatedInput | null, AdditionalProps>, payment: Form<State, {required: true, notCheckedErrorMessage: string} | {required: false}, boolean, AdditionalProps>}>
```

Reply:
```suggestion
        label='Für diese ehrenamtliche Tätigkeit wurde eine Aufwandsentschädigung gewährt, die über den jährlichen Freibetrag hinaus geht (840 Euro Ehrenamtspauschale bzw. 3000 Euro Übungsleiterpauschale).'
```

Reply:
@f1sh1918 I agree, that there should not be a typescript error when editing a label. Not sure, if this might have been a caching-problem in your local setup, as the CI checks seem to run smoothly?